db/memtable_list.h

```diff
@@ -267,6 +279,8 @@ class MemTableList {
   }
 
  private:
+  static bool atomic_flush_commit_in_progress_;
```

Comment:
Using a static won't work when multiple DB instances run in the same process (actually that includes ZippyDB). We should move ownership of this to a DB-wide structure.

Reply:
You are right. Will fix.